src/components/containers/Status/CurrentValBox.js

```diff
@@ -0,0 +1,35 @@
+import { BsArrowUp } from "react-icons/bs";
+let CurrentValBox = () => {
+  return (
+    <>
+      <div className="flex flex-col items-center mt-10">
+        <div className="flex flex-col bg-[#BFFA00] rounded max-w-[90%] h-[40%] rounded-[0.625rem] ">
+          <div className="flex px-[3vw] py-[1vw] lg:py-[.7vw] ">
+            <div className="flex items-baseline">
+              <h1 className="text-[3rem] font-bold">21°C</h1>
+              <BsArrowUp className="text-[2rem] "></BsArrowUp>
+            </div>
+
+            <div className="flex flex-col mt-[.7rem] ml-[1rem] ">
+              <h3 className="font-bold">Your Greenhouse</h3>
+              <h5 className="text-[.85rem]">at 10:08 10/12/2023</h5>
```

Comment:
data, humidity, co2, temperature should be parameterized.

Reply:
ok I will add it